### PR TITLE
New package: The-Powder-Toy-94.1

### PR DESCRIPTION
**Pull request by singul0**

### CODE REVIEW

File: srcpkgs/ThePowderToy/template [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+maintainer="Illia Shestakov <ishestakov@airmail.cc>"
+license="GPL-3.0-or-later"
+homepage="https://powdertoy.co.uk/"
+distfiles="https://github.com/The-Powder-Toy/The-Powder-Toy/archive/v${version}.tar.gz"
+checksum=9770c0732c97e4e89a1e53330c969e9675595f78e6673c519c3724d6df591e72
+CPPDEFINES=IGNORE_UPDATES
+
+do_build() {
+	case "$XBPS_TARGET_MACHINE" in aarch64*|arm*|ppc*)
+		export SCONS_ARGS="--no-sse";;
```
Comment (Chocimier) [inline srcpkgs/ThePowderToy/template]: Do setting `make_build_args` for default do_build work?

File: srcpkgs/ThePowderToy/template [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	case "$XBPS_TARGET_MACHINE" in
+	x86_64*)
+		vbin build/powder64 powder;;
+	i686*)
+		vbin build/powder powder;;
+	*)
+		vbin build/powder-legacy powder;;
+	esac
+	vdoc README.md
+	vlicense LICENSE
```
Comment (Chocimier) [inline srcpkgs/ThePowderToy/template]: We do not include GPL into package.

File: srcpkgs/ThePowderToy/template [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Template file for 'ThePowderToy'
+pkgname=ThePowderToy
```
Comment (Chocimier) [inline srcpkgs/ThePowderToy/template]: I would name it The-Powder-Toy, like in urls.

**Comment by singul0** [comment]
@Chocimier thank you for the review

File: srcpkgs/The-Powder-Toy/template [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+short_desc="Falling sand physics sandbox, simulates air pressure, velocity & heat"
+maintainer="Illia Shestakov <ishestakov@airmail.cc>"
+license="GPL-3.0-or-later"
+homepage="https://powdertoy.co.uk/"
+distfiles="https://github.com/The-Powder-Toy/${pkgname}/archive/v${version}.tar.gz"
+checksum=9770c0732c97e4e89a1e53330c969e9675595f78e6673c519c3724d6df591e72
+CPPDEFINES=IGNORE_UPDATES
+
+case "$XBPS_TARGET_MACHINE" in aarch64*|arm*|ppc*)
+	export make_build_args="--no-sse";;
```
Comment (Johnnynator) [inline srcpkgs/The-Powder-Toy/template]: No need for export, also pls make this a whitelist with only `x86_64` and `i686`, currently this wouldn't disable SSE for `riscv` or `mips`

File: srcpkgs/The-Powder-Toy/template [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+esac
+
+do_install() {
+	case "$XBPS_TARGET_MACHINE" in
+	x86_64*)
+		vbin build/powder64 powder;;
+	i686*)
+		vbin build/powder powder;;
+	*)
+		vbin build/powder-legacy powder;;
```
Comment (Duncaen) [inline srcpkgs/The-Powder-Toy/template]: Maybe set the output option instead https://github.com/The-Powder-Toy/The-Powder-Toy/blob/master/SConscript#L571

Comment (singul0) [reply srcpkgs/The-Powder-Toy/template]: You're right, it works

**Pull request merged**